apps/sms/views/shared/test/unit/settings_test.js

```diff
@@ -222,9 +235,9 @@ suite('Settings >', function() {
       navigator.mozMobileConnections = [{ iccId: 'SIM 1' }, { iccId: null }];
       Settings.init();
 
-      // Four calls for
-      // maxConcatenatedMessages/mmsSizeLimitation/mmsServiceId/smsServiceId
-      sinon.assert.callCount(navigator.mozSettings.createLock, 4);
+      // Five calls for maxConcatenatedMessages/mmsSizeLimitation
+      // sendReadReport/mmsServiceId/smsServiceId/
+      sinon.assert.callCount(navigator.mozSettings.createLock, 5);
```

Comment:
I think it's testing things that are too internal; I don't think we test the external behavior with these checks. Remember the goal of tests: test the code from the outside. Then if the internal code can change but not the external behavior/interface, the test should not change.

In this case I think it's messy to test the count of createLock..